packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3314,6 +3318,7 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
 
   bool get _hasFocus => widget.focusNode.hasFocus;
   bool get _isMultiline => widget.maxLines != 1;
+  bool get _canTapOutsideFocus => widget.canTapOutsideFocus;
```

Comment:
nit: This doesn't need to be stored.

Reply:
I made many unnecessary changes before, but the change seems to be simple. One question, I wouldn't have to store this value for when it is in another `TextField `(or variants), or I can pass it as:
`onTapOutside: _hasFocus ||  widget.canTapOutsideFocus ? widget.onTapOutside ?? _defaultOnTapOutside : null`?

Reply:
```dart
onTapOutside: _hasFocus || widget.canTapOutsideFocus ? widget.onTapOutside ?? _defaultOnTapOutside : null
```

Should work.